parser.y

```diff
@@ -2360,6 +2361,7 @@ KeyOrIndexOpt:
 |	KeyOrIndex
 
 ColumnKeywordOpt:
+	/* empty */ %prec empty
```

Comment:
why should we put a prec here?

Reply:
`ColumnKeywordOpt` conflicts with `placement`. No prec will skip the conflict resolution of goyacc, however.